apps/bare-expo/android/app/src/main/java/dev/expo/payments/MainActivity.java

```diff
@@ -38,9 +35,6 @@ protected ReactRootView createRootView() {
       @Override
       protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        // SplashScreen.show(...) has to be called after super.onCreate(...)
-        // Below line is handled by '@expo/configure-splash-screen' command and it's discouraged to modify it manually
-        SplashScreen.show(activity, SplashScreenImageResizeMode.COVER, ReactRootView.class, false);
       }
     };
```

Comment:
We don't need this override at all then 🤔

Reply:
That's what's being proposed in this PR. Regexing MainActivity isn't safe and we need to move away from it.

Reply:
yeah we can just delete this onCreate override entirely here